.github/workflows/docker.yml

```diff
@@ -11,6 +11,7 @@ jobs:
       matrix:
         ruby_version: [
           2.6.7,
```

Comment:
Does this need to stick around?

Reply:
for now yes, until I finish bumping all repos that use 2.6.7 to 2.6.8